packages/amplify-e2e-tests/src/__tests__/schema-iterative-rollback-1.test.ts

```diff
@@ -0,0 +1,63 @@
+import * as path from 'path';
+import {
+  createNewProjectDir,
+  initJSProjectWithProfile,
+  deleteProject,
+  deleteProjectDir,
+  amplifyPushIterativeRollback,
+  getDDBTable,
+  getBackendAmplifyMeta,
+  addApiWithSchema,
+  addFeatureFlag,
+  amplifyPush,
+  updateApiSchema,
+  getTableName,
+  getNestedStackID,
+  cancelIterativeAmplifyPush,
+} from 'amplify-e2e-core';
+
+// 30-45min
+describe('Iterative Rollback - add 2 @keys ', () => {
+  let projectDir: string;
+
+  beforeAll(async () => {
+    projectDir = await createNewProjectDir('iterativeRollback');
+    await initJSProjectWithProfile(projectDir, { disableAmplifyAppCreation: false });
```

Comment:
Does this test need amplify app creating?

Reply:
No this leftover when I tested this flag with pull